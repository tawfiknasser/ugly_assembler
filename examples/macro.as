add r1,r2
mcr im_macro_1
add r0,r1
endmcr

mcr im_macro
add r3,r4 
add r1,r2
add r1,r2
endmcr

im_macro
im_macro_1
im_macro
