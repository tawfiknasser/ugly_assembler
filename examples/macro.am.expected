add r1,r2
add r3,r4 
add r1,r2
add r1,r2
add r0,r1
add r3,r4 
add r1,r2
add r1,r2
